doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
The command line generation is made by calling `python MY_COMMAND --help`.

It is done in `doc/sphinxext/gen_commands.py` l.68

Thus the new argument should be in the documentation all right.